demo/bulk_insert/bulk_insert.py

```diff
@@ -168,17 +167,19 @@ def process_entities(self, expected_col_count):
                 if row[0] in NODE_DICT:
                     print("Node identifier '%s' was used multiple times - second occurrence at %s:%d"
                           % (row[0], self.infile.name, self.reader.line_num))
+                    exit(1)
```

Comment:
raise, might be just as fine, what do we do with the graph which was created up to this point?